test/test_external.py

```diff
@@ -252,11 +252,11 @@ def test_conversational_in_blocks(self):
         client = TestClient(app)
         response = client.post(
             "/api/predict/",
-            json={"session_hash": "foo", "data": ["Hi!", None], "fn_index": 0},
+            json={"session_hash": "foo", "data": ["Hi!"], "fn_index": 0},
```

Comment:
Looks like the classification of this model has changed, now its no longer a chatbot